UNIX/cleanup.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# For a description of the config variables set/used here,
+# see automation/Windows/build-and-test.bat.
```

Comment:
Is the reference to `Windows/build-and-test.bat` in the unix script intentional?